src/lein_monolith/ansi.clj

```diff
@@ -0,0 +1,13 @@
+(ns lein-monolith.ansi
+  (:require
+    [puget.color.ansi :as ansi]
+    [puget.printer :as puget]))
+
+
+(defn maybe-sgr
```

Comment:
If you move it to the util ns as above, a more general name here would be better. How about `color-text`? I don't think the configurability needs to be called out in the function name.